### PR TITLE
Dehardcode weather

### PR DESCRIPTION
**Pull request by amyeallen1**
Draft pull request for group input re: failing test.

### CODE REVIEW

File: geojson_modelica_translator/system_parameters/system_parameters.py [hunk 1]
```diff
@@ -43,11 +43,13 @@ class SystemParameters(object):
     PATH_ELEMENTS = [
         {"json_path": "$.buildings.default.load_model_parameters.spawn.idf_filename"},
         {"json_path": "$.buildings.default.load_model_parameters.spawn.epw_filename"},
+	    {"json_path": "$.buildings.default.load_model_parameters.rc.mos_weather_filename"},
```
Comment (nllong) [inline geojson_modelica_translator/system_parameters/system_parameters.py:46]: nice catch! This one has bit me in the past!

Comment (amyeallen1) [reply geojson_modelica_translator/system_parameters/system_parameters.py:46]: Thanks for taking a look at it!

**Comment by amyeallen1** [comment]
The problems I was running into before are now resolved and so this PR is ready for review for inclusion. @nllong @vtnate Thanks for your help with this!

**Review by nllong — APPROVED** [review]
looks good. I tweaked a few files, but overall the changes were spot on.

**Comment by amyeallen1** [comment]
Thanks, Nick!

**Pull request merged**